src/xo-app/settings/remotes/index.js

```diff
@@ -23,34 +28,49 @@ const remoteTypes = {
 }
 
 class AbstractRemote extends Component {
+  _changeUrlElement = (value, element) => {
+    let { remote } = this.props
+    remote = {...remote}
```

Comment:
What about xo-remote-parser 3, is someone working on it, if not, do you want me to ?